### DISCUSSION TITLE
Search in Play Store was "difficult" (SEO?)

### DISCUSSION THREAD

**Discussion by KalobTaulien** [CAWSCIT/ca-meeting-finder-issue-tracker#18] [Feedback]
This is anecdotal as I do not have an Android phone. 

Some new users were searching for "CA App", "cocaine anonymous", and "CA meeting finder" and claimed they couldn't find the app. 

Are we able to add tags, or increase the app description to include some keywords that may help with app discovery?

**Comment by sgrant-SFVCA** [comment]
Unfortunately, this problem exists on the App Store as well.  The problem is the app name as C.A. Rather than just CA.   It also makes it more difficult to locate the app on the iPhone.   

If the App could be renamed or an alias be used, the user experience would be improved.   Searching for CA Meeting or CA Meetings does work but…

Stuart
